frontend/src/pages/MetadataByType/index.jsx

```diff
@@ -30,6 +27,7 @@ const MetadataTypesViewContainer = styled.div`
 `;
 
 const LeftPane = styled.div`
+  height: 70vh;
```

Comment:
`LeftPane` is the list of metadata types. It should either be a min-height, or no height specification at all, to prevent it from getting cut off like in the screenshot.
```suggestion

```

Reply:
Also, I've tested this + your changes in Firefox :)

Reply:
Liam didn't want it to stretch downwards. That's why it got broken in the first place.

Reply:
Ah I see

Reply:
So, what we doin

Reply:
We can keep the styling for all I care, but I think we should add the filtering fixes I suggested.